proposals/lip-Introduce_cross-chain_update_mechanism.md

```diff
@@ -0,0 +1,620 @@
+
+When a sidechain is started and registered, the sidechain developers might decide to not activate the sidechain straight away (maybe to do further testing).
+It could happen then (intentionally or not) that an old block header (almost 30 days old) is submitted to the mainchain to activate the sidechain.
+This could result in the sidechain being punished for liveness failure very soon after the activation (maybe only a few minutes later). 
+To prevent this issue (and without any significant drawbacks) the first cross-chain update to be submitted on mainchain must contain a certificate less that 15 days old.
+The sidechain has therefore at least 15 days to submit the next cross-chain update to the mainchain and start the regular posting of cross-chain updates.
+
+
+## Specification
+
+The Interoperability module supports two commands used to certify the state of another chain. 
+Those commands have `moduleID = MODULE_ID_INTEROPERABILITY`. 
+One of them, meant to be posted on the mainchain, has `commandID = COMMAND_ID_MAINCHAIN_CCU`, 
+while the other, meant to be posted on sidechains, has `commandID = COMMAND_ID_SIDECHAIN_CCU`.
+
+
+### Constants and Notations
+
+The following constants are used throughout the document:
+
+| Name          | Type    | Value       |
+| ------------- |---------| ------------|
+| **Interoperability Constants** |||
+| `MODULE_ID_INTEROPERABILITY` | uint32 | 64     |
+| `STORE_PREFIX_CHAIN_DATA`    | bytes  | 0x8000 |
+| **Interoperability Command and Cross-chain Command IDs** |||
+| `COMMAND_ID_SIDECHAIN_CCU`             | uint32 | 2 |
+| `COMMAND_ID_MAINCHAIN_CCU`             | uint32 | 3 |
+| `CROSS_CHAIN_COMMAND_ID_CCU_RECEIPT`   | uint32 | 1 |
+| **Chain Status** |||
+| `CHAIN_REGISTERED` | uint32 | 0 |
+| `CHAIN_ACTIVE`     | uint32 | 1 |
+| `CHAIN_TERMINATED` | uint32 | 2 |
+| **Message Tags** |||
+| `MESSAGE_TAG_CERTIFICATE ` | bytes | "LSK_CE_" ASCII-encoded |
+
+Several of those constants are shared with the other LIPs defining the Interoperability module and all of the needed constants for the Interoperability module are defined in [LIP "Introduce Interoperability module"][base-interoperability-LIP]. 
+That LIP should be considered correct if a value stated here differs.
+
+
+#### uint32be
+
+`uint32be(x)` returns the big endian uint32 serialization of an integer `x`, with `0 <= x < 2^32`. 
+This serialization is always 4 bytes long.
+
+
+### Parameters Schema
+
+Both commands will use the following `params` schema:
+
+```java
+crossChainUpdateTransactionParams = {
+    "type": "object",
+    "required": [
+        "sendingChainID", 
+        "certificate", 
+        "validatorsUpdate",
+        "inboxUpdate"
+    ],
+    "properties": {
+        "sendingChainID": {
+            "dataType": "uint32",
+            "fieldNumber": 1
+        },
+        "certificate": {
+            "dataType": "bytes",
+            "fieldNumber": 2
+        },	
+        "validatorsUpdate": {
```

Comment:
Just a reminder: If we update the chain account schema in the main interoperability LIP, it would be better to update this schema as well, and remove the nesting for the  "activeValidatorsUpdate", "newCertificateThreshold" properties

Reply:
yes, changed.